CMakeLists.txt

```diff
@@ -1261,6 +1261,14 @@ install(
   DESTINATION ${PKG_CONFIG_INSTALL_DIR}
 )
 
+# Export build tree to ease development and debugging
+target_include_directories(allegro INTERFACE
```

Comment:
Could you take a look at `cmake/Common.cmake` `add_our_library` and see if this should go there instead (alongside the other `target_include_directories` call)?

Reply:
Ok, I think I get what's going here. We only add it to allegro because addons link to allegro, so we don't really need to set it for all of them as well.

I think then what's missing is to make this conditional on ALLEGRO_MONOLITH not being defined (and then call this again for the monolith).

Does that make sense?

Reply:
If nothing else, this needs to be disabled if WANT_MONOLITH is on, since otherwise it gives an error that the `allegro` target doesn't exist.